api/envoy/config/filter/http/grpc_http1_reverse_bridge/v2alpha1/config.proto

```diff
@@ -24,3 +24,10 @@ message FilterConfig {
   // simple binary encoded protobuf.
   bool withhold_grpc_frames = 2;
 }
+
+// gRPC reverse bridge filter configuration per virtualhost/route/weighted-cluster level.
+message FilterConfigPerRoute {
+  // If true, disables gRPC reverse bridge filter for this particular vhost or route.
+  // If disabled is specified in multiple per-filter-configs, the most specific one will be used.
+  bool disabled = 1;
```

Comment:
@mattklein123 @rshriram @envoyproxy/api-shepherds where did we end up with just having a generic `disable` per filter? It seems kind of crazy we need to add docs, tests, etc. for every single filter we want to have a disable/pass-thru on at the per-route level.

Reply:
I agree we should do this, but I'm not sure we should block this PR on that.

Reply:
Sure, let's merge, I've opened https://github.com/envoyproxy/envoy/issues/8669 to track.